gpMgmt/bin/go-tools/test/integration/init_cluster/init_cluster_cleanup_test.go

```diff
@@ -0,0 +1,469 @@
+package init_cluster
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/greenplum-db/gpdb/gp/cli"
+	"github.com/greenplum-db/gpdb/gp/test/integration/testutils"
+)
+
+func TestInitClusterCleanup(t *testing.T) {
+
+	t.Run("check if the coordinator is stopped whenever an error occurs", func(t *testing.T) {
+
+		var valueSegPair []cli.SegmentPair
+		var okSeg bool
+		var value cli.Segment
+		var ok bool
```

Comment:
Combine in single declaration 
```suggestion
		var (
	valueSegPair []cli.SegmentPair
	 okSeg bool
	 value cli.Segment
	 ok bool )
```

Reply:
Any specific reason you are declaring variables beforehand?

Reply:
Done